core/config.py

```diff
@@ -38,7 +42,7 @@ def populate_cache(self):
             'aliases': {},
             'blocked': {},
             'notification_squad': {},
-            'subscriptions': {}
+            'subscriptions': {},
```

Comment:
Add closures here as well in the initial schema so that you won't need to do `self.config.get('closures', {})